apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerUtils.java

```diff
@@ -3023,13 +3017,13 @@ public static String retrieveIcdCodeDisplay(String icdCode) {
     }
 
     // log which NDC codes we couldn't find a match for in our downloaded NDC file
-    if (!drugCodeLookupMissingFailures.contains(icdCode)) {
-      drugCodeLookupMissingFailures.add(icdCode);
+    /*  if (!drugCodeProvider.drugCodeLookupMissingFailures.contains(icdCode)) {
+      drugCodeProvider.drugCodeLookupMissingFailures.add(icdCode);
       LOGGER.info(
           "No ICD code display value match found for ICD code {} in resource {}.",
           icdCode,
           "DGNS_CD.txt");
-    }
+    } */
```

Comment:
this looks like it is incorrect

Reply:
i took this out dunno since there is a check inside the drugutils